src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/_help.py

```diff
@@ -135,13 +135,24 @@
             short-summary: Take ownership of an existing virtual machine in Azure DevTest Lab.
             parameters:
                 - name: --ids
-                  short-summary: One or more resource IDs (space delimited). If provided, no other arguments should be specified
+                  short-summary: One or more resource IDs (space delimited). If provided, no other arguments should be specified.
+                                 If --ids and --name both are omitted then provide --resource-group and --lab-name to claim any available VM.
```

Comment:
Recommended text: "Space separated list of VM IDs to claim."'

I would omit the second line because your example already covers that. You could also clarify in the long or short summary for the command that you can claim a specific VM from the pool or any that is available in the lab.